mpqp/qasm/open_qasm_2_and_3.py

```diff
@@ -740,3 +760,333 @@ def remove_include(qasm_code: str) -> str:
         else:
             replaced_code += line + "\n"
     return replaced_code
+
+
+@typechecked
+def convert_instruction_3_to_2(
```

Comment:
_I put here the comment because I didn't how where to put it and I wanted to be sure we can answer below._

When there is noise in the Braket circuit, and we would like to export that circuit to OpenQASM3, the noise models will be encoded in the OpenQASM3 using the `pragma` keyword (see example below).

I see two possible options to handle that : 
1. We grab the noise models directly from the Braket circuit before the conversion to OpenQASM3, to set it separately later in the QCircuit
2. We parse and analyse the pragmas to generate the right noisemodels in mpqp

In both cases, we can typically use a NoiseModel.from_other_language("str"(pragma) or BraketInstruction (which is a noise operator)).

To be discussed, if second choice, this will have impact on this PR.